### PR TITLE
decription change for usedConnections and queuedRequests

### LINKED ISSUE #5674: Updating mpMpetrics useConnections and queuedRequests description
Part of LG-26 change. Just message changes and a minor fix in code comment.

### PR DESCRIPTION
**Pull request by fwji**
fixes #5674

### CODE REVIEW

**Comment by fwji** [comment]
#build

File: dev/com.ibm.ws.microprofile.metrics/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops [hunk 1]
```diff
@@ -120,8 +120,8 @@ connectionpool.connectionHandles.description=The number of connections that are
 connectionpool.freeConnections.description=The number of managed connections in the free pool.
 connectionpool.waitTime.total.description=The total wait time on all connection requests since the start of the server.
 connectionpool.inUseTime.total.description=The total time that all connections are in use since the start of the server.
-connectionpool.usedConnections.total.description=The total number of used connections.
-connectionpool.queuedRequests.total=The total number of requests that waited in the queue before a connection was granted.
+connectionpool.usedConnections.total.description=The total number of connection requests that waited because of a full connection pool or did not have to wait since the start of the server. Any connections that are currently in use are not included in this total.
+connectionpool.queuedRequests.total=The total number of connection requests that had to wait for a connection because of a full connection pool since the start of the server.
```
Comment (alexsm82) [inline dev/com.ibm.ws.microprofile.metrics/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops]: Should this be connectionpool.queuedRequests.total.description?

**Comment by fwji** [comment]
#build

**Comment by fwji** [comment]
3 failed test are not monitor or mpMetrics related, and hence safe to merge

**Pull request merged**